pkg/monitor/event_test.go

```diff
@@ -103,7 +104,7 @@ func Test_recordAddOrUpdateEvent(t *testing.T) {
 		}
 		t.Run(tt.name, func(t *testing.T) {
 			for _, event := range tt.args.kubeEventList.Items {
-				recordAddOrUpdateEvent(tt.args.ctx, tt.args.m, tt.args.client, tt.args.reMatchFirstQuote, tt.args.significantlyBeforeNow, &event)
+				RecordAddOrUpdateEvent(tt.args.ctx, tt.args.m, tt.args.client, tt.args.reMatchFirstQuote, tt.args.significantlyBeforeNow, &event, duplicateevents.DuplicateEventThreshold)
```

Comment:
Never noticed before but does this test not assert anything? Just checks that it runs and presumably a human is watching the console? Could we get some assertions in here so we get notified if something breaks this.

Reply:
ah you're right.  I probably just used it to step through that function.  I can tweak it to check for some kind of specific output.